ci/test_rockylinux9.sh

```
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+#
+# Test GoSnowflake driver in Rocky Linux 9
+# NOTES:
+#   - Go version to be tested should be passed in as the first argument, e.g: "1.24". If omitted 1.24 will be assumed.
```

I don't think we should assume 1.24. It's going to be deprecated soon. Let's fail fast.